docs/software/container-development-guide.md

```diff
@@ -33,7 +37,86 @@ Creating a New Container Image
 
     Replacing `<PACKAGE>` with the name of the RPM you'd like to provide in this container image
 
-### Prepare the DockerHub repository ###
+1. Copy over `.github/workflows/build-container.yml` from a previous docker build GitHub repo
+   (e.g., <https://github.com/opensciencegrid/docker-frontier-squid>) to the same path in your new git repo.
+1. Update the `if: github.repository` line in the new copy of `build-container.yml` to match the new git repo name.
+   (This should keep the `opensciencegrid/` org prefix, and the new git repo name should start with `docker-`.)
+1. Also update the `repository:` line (in `build-container.yml`) under `uses: docker/build-push-action@v1` to match the
+   corresponding DockerHub repo name.
+   (This also should keep the `opensciencegrid/` org prefix, and the repo name is generally the same as the git repo,
+   except without the `docker-` prefix.)
+1. If there are any other references to the old repo name (e.g., `checkout frontier-squid`),
+   replace these with the new repo name.
+1. Give write permissions to the "osg-bot" user for this GitHub repo, navigating to:
+   1. "Settings"
+   1. "Manage access"
+   1. "Invite teams or people"
+   1. Search for and select "osg-bot"
+   1. Choose the "Write" role, and click the button to Add osg-bot to the repo.
+   (The osg-bot user needs this permission in order to trigger automatic builds.)
+1. Ask the Software Manager to set up `DOCKER_USERNAME` and `DOCKER_PASSWORD` secrets for this new repo.
+
+#### Set up repository dispatch from docker-software-base (GitHub Actions) ####
+
+1. Edit the
+   [GitHub Actions workflow for docker-software-base](https://github.com/opensciencegrid/docker-software-base/blob/master/.github/workflows/build-container.yml),
+   and add the new GitHub repo name to the `dispatch-repo:` list (under `jobs:` `dispatch:` `strategy:` `matrix:`).
+1. Make a Pull Request for your change.
+
+#### Prepare the DockerHub repository (GitHub Actions) ####
```

Comment:
```suggestion
#### Prepare the Docker Hub repository ####
```

Also, move this section above the repo dispatch setup

Reply:
>Also, move this section above the repo dispatch setup

The reason i thought the repo dispatch setup would go first is that that is still part of the github setup (though for the docker-software-base repo), before moving on to docker hub setup.  But anyway i've updated it.

Reply:
I'm just worried that the update to `docker-software-base` could get merged, triggering a docker push from the new GitHub repo before the Docker Hub repo is ready.